src/main/resources/swagger/openapi.yaml

```diff
@@ -654,6 +654,10 @@ components:
           type: array
           items:
             type: string
+            example:
+              - 630d31c3-381e-488d-b639-ce5d047a0142
+              - 'dockstore.org:630d31c3-381e-488d-b639-ce5d047a0142'
+              - 'bio.tools:630d31c3-381e-488d-b639-ce5d047a0142'
```

Comment:
Any reason this is formatted differently between the two files?

Reply:
No, but openapi.yaml is generated by auto-conversion from the swagger.yaml (using https://www.npmjs.com/package/swagger2openapi ) as a convenience so they should be equivalent.